string objecttag
float64 object_x
float64 object_y
int64 mid_bb_angle
